### PR TITLE
GOODS-6428 Add svgs for reason codes

### PR DESCRIPTION
**Pull request by aahill50**
Adds 4 svgs, one to represent each reason code

### CODE REVIEW

File: src/circle-exclamation.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-circle-exclamation" viewBox="0 0 250 250" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (hellatan) [inline src/circle-exclamation.svg]: don't we have one of these already?

Comment (aahill50) [inline src/circle-exclamation.svg]: I don't see one

File: src/circle-contact-info.svg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		    c15.9,0.1,31.9,0.1,47.4,0.1c23,0,37.2-0.1,37.4-0.1l4.8,0l-1.1-14.4L129.4,154.8z M88.3,160.8c-13.9,0-28.2,0-42.5-0.1
+		    c0.4-3.4,0.9-6.7,1.4-7.8l0.1-0.3c0,0,0.9-2.1,2.9-2.8l14.2-5.7l5.9-2.6c1.1-0.4,5.9-2.7,7.3-8.8l0.1-0.3c0.1-0.9,0.3-3.2-0.6-5.1
+		    c-0.7-1.3-1-1.7-2-2.9c-0.4-0.5-4.6-5.8-5.5-13l0-0.1c0-0.1-2.2-15.1,5.7-19.9c0.4-0.2,3.7-1.9,8.3-1.9c2.9,0,5.7,0.7,8.3,2
+		    l0.4,0.2c0.1,0,7.5,3.7,5.2,20l0,0.1c0,0.1-0.9,7.1-5.6,12.7c-0.1,0.1-0.1,0.2-0.2,0.2c-0.5,0.6-1.4,1.5-2,2.7l-0.2,0.4
+		    c-0.2,0.6-0.9,2.7-0.1,5.3c0.4,1.4,1.9,5.9,7.6,8.4l7.6,3.5l11.5,4.6c0,0,3.5,1.5,4.3,6.5l0.3,4.6
+		    C114.8,160.7,103.5,160.8,88.3,160.8z"
+        />
+        <rect x="144.3" y="97.9" fill="#010101" width="66.7" height="9"/>
+        <rect x="144.3" y="121.6" fill="#010101" width="66.7" height="9"/>
+        <rect x="144.3" y="145.3" fill="#010101" width="66.7" height="9"/>
```
Comment (hellatan) [inline src/circle-contact-info.svg]: the `fill` attribute should be removed. I don't think you need to target these specifically, right? The fill should be the same as every other element in this svg?

File: src/circle-contact-info.svg [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-circle-contact-info" viewBox="0 0 250 250" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g>
```
Comment (hellatan) [inline src/circle-contact-info.svg]: i think all of these `<g>` tags in all the files can be removed, is that right @nameofname?

File: src/circle-contact-info.svg (hunk 2 above)
Comment (aahill50) [inline src/circle-contact-info.svg]: Ahh I didn't see this, I removed the fill from the other elements in this svg, just missed these

File: src/circle-exclamation.svg (hunk 1 above)
Comment (hellatan) [inline src/circle-exclamation.svg]: look for `exclamation-circle`

Comment (aahill50) [inline src/circle-exclamation.svg]: Hmm weird, I just looked at the list of svgs and it shows up there, but it doesn't show up in the sample list that gets generated when you compile them all

Comment (aahill50) [inline src/circle-exclamation.svg]: Ok I think I just didn't have the latest version of the repo pulled down before I started on these

**Comment by aahill50** [comment]
@hellatan removed extra `<g>` tags, `fill` properties and duplicate exclamation svg

**Comment by hellatan** [comment]
@aahill50 can you rebase on top of the latest master? I merged in #84 so when you publish next, it should get those updates.

File: list.json [hunk 4]
```diff
@@ -26,6 +26,9 @@
   "circle-arrow-right.svg",
   "circle-cancel.svg",
   "circle-check.svg",
+  "circle-contact-info.svg",
+  "circle-dollar.svg",
```
Comment (hellatan) [inline list.json]: lolz...there is a "sales" icon and another "upload-failed" for the exclamation point. I don't know where either one is currently being used, however. But I prefer the `circle-dollar` name instead.

Comment (hellatan) [inline list.json]: "sales" looks like just here - https://github.com/search?q=dibs-vg+sales&type=Code&utf8=%E2%9C%93

Comment (aahill50) [inline list.json]: hmmm, yeah i did a quick search for "dollar" and "currency" to see if it existed already, didn't bother to scroll through everything... Yeah I think `circle-dollar` is an appropriately generic name

**Comment by aahill50** [comment]
rebased

**Comment by hellatan** [comment]
@aahill50 ⬇️ 〰️ tasks, please

**Comment by aahill50** [comment]
v1.38.0 published

**Pull request merged**